tests/acceptance/features/apiGraph/enforcePasswordPublicLink.feature

```diff
@@ -0,0 +1,33 @@
+@api @env-config
+Feature: enforce password on public link
+  As a user
+  I want to enforce password on upload, edit or contribute public link share
+  So that other users with password can make changes on files
```

Comment:
```suggestion
  I want to enforce passwords on public links shared with upload, edit, or contribute permission
  So that the password is required to make changes to the shared resources
```